libnavui-dropin/src/main/res/layout/mapbox_action_free_drive_layout.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:orientation="vertical"
+    tools:parentTag="android.widget.LinearLayout">
+
+    <com.mapbox.navigation.ui.maps.camera.view.MapboxRecenterButton
```

Comment:
@abhishek1508 Should I also implement new, squared Recenter Button  as designed in Figma?

Reply:
Yeah we can do that either in this PR or a separate one

Reply:
Separate PR it is.